src/client/components/Text/ErrorMessage.tsx

```diff
@@ -32,7 +32,7 @@ export const ErrorMessage: FunctionComponent<ErrorMessageProps> = (
 	const { children } = props;
 	return (
 		<Rectangle>
-			<SadFace />
+			<img src={SadFace} alt="Sad face" />
```

Comment:
Does this have any advantages/disadvantages of native svg component?

Reply:
It's more disadvantaged, actually, but it was a much quicker solution at the time. The downside of using SVGs inside img tags:

- Must be loaded _after_ the dom has finished loading, leading to pop-in and delaying TTFCP
- Cannot dynamically change SVG properties like fill color, stroke, etc.
- The size of most of the svgs we use is minimal because they are simple vector art and optimized anyway.

Benefits:
- img tags don't increase the js bundle size (much)
- img tags are easier to reason about for people newer to frontend (most tutorials on the internet are for img tagged svgs)
- Handle flexbox auto sizing well by default

Seeing the above considerations, I really like inline svg's. The reason I went with this here is due to the third bullet under benefits of img tagged svgs. I didn't have to worry about the SVG tag's width and height causing it to take up the whole screen. Also didn't try, so not _sure_ that would happen, but it's an issue we've run into before.